configs/mask2former/mask2former_r50_8xb2-lsj-50e_coco.py

```diff
@@ -56,6 +56,17 @@
     dict(type='PackDetInputs')
 ]
 
+test_pipeline = [
```

Comment:
Why this modification is necessary？

Reply:
@jshilong 
Because the origin config can not launch experiment successfully.
I understand that this config is for instance segmentation, however, the config is inherited from coco-panoptic config.